src/angularComponent/ngcOmniboxController.js

```diff
@@ -200,6 +200,19 @@ export default class NgcOmniboxController {
     }
   }
 
+  /**
+   * Removes a particular suggestion item frmo the list of selected items.
```

Comment:
1) There's a typo in this line.

2) What does this do if we're in single-select mode?